CIP-DRep-Credential/README.md

```diff
@@ -0,0 +1,393 @@
+---
+CIP: ???
+Title: On-Chain DRep Credential & Extended Governance NFTs
+Status: Draft
+Category: Tokens
+Authors:
+  - "Derrick Oatway <alpine@malamaproject.org>"
+Implementors:
+  - DRep Collective
+Discussions:
+  - "https://github.com/cardano-foundation/CIPs/pull/1007"
+Created: "2025-03-01"
+License: CC-BY-4.0
+---
+
+## Abstract
+
+> "**Poets are the unacknowledged legislators of the world.**"  
+> — Percy Bysshe Shelley, _A Defence of Poetry_ (1821)
+
+This CIP introduces structured **on-chain** metadata standards to enhance transparency, accountability, and effectiveness in Cardano’s governance. Building on **CIP-1694**, it provides new mechanisms for Delegated Representatives (DReps) and delegators to record, discover, and analyze governance data in a standardized way. Leveraging the **CIP-68** datum-based NFT model, it defines three specialized token types:
+
+1. **DRep Credential** – An on-chain credential NFT referencing both on- and off-chain metadata (per **CIP-119**), with optional fields (e.g., roles, sectors) for DReps to self-report.
+2. **Ballot Note** – A verifiable record capturing a DRep’s vote and accompanying rationale.
+3. **Endorsement** – An independently minted NFT serving as decentralized social proof from a DRep’s supporters.
+
+Where large content is required—like detailed profiles, mission statements, or rationales—it can be stored off-chain and anchored via **CIP-119** or **CIP-108**, minimizing on-chain storage. These tokens do not replace the core mechanics of CIP-1694; rather, they offer an **optional**, more transparent framework for tracking DRep credentials, roles, votes, and social endorsements—ultimately enabling delegators and the community to make better-informed decisions.
+
+## Motivation: why is this CIP necessary?
+
+> "**Every man is guilty of all the good he did not do.**"  
+> — François-Marie Arouet (Voltaire), 18th century
+
```

Comment:
```suggestion
```
Again I could swear these were removed in an earlier commit; did they creep back in with a merge in your fork @Alpine-Oracle?  My apologies if one of the 100 discussions in @Crypto2099's initial review established that 2 of the original epigraphs should be left in the document 😅